.github/workflows/docker-slurm.yml

```diff
@@ -9,12 +9,15 @@ on:
 
 jobs:
   docker:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu2204-64c-256g-2040ssd
```

Comment:
What problems were you seeing with a less powerful runner? Was it not able to handle the volume of `lfs`? Would it time out?

Reply:
The multi-platform build now needs to build both the `amd64` and `arm64` images.  Additionally, the `arm64` build must be done under emulation mode (with qemu), which runs very very slowly.  So, the two builds are sharing resources at the same time, and one of them runs extremely slowly.  So I have yet to find a runner powerful enough to finish within the 6 hour time limit.  If the latest one fails, we'll have to figure out what to do since we won't be able to build the multi-platform image in GHA.